src/app/home/dashboard/dashboard.component.html

```diff
@@ -46,7 +47,7 @@ <h3><fa-icon icon="laptop"></fa-icon>
       <mifosx-amount-collected-pie></mifosx-amount-collected-pie>
 
     </div>
-    
+    <div *ngIf="!supersetEnable"><mifosx-superset></mifosx-superset></div>
```

Comment:
This seems wrong to me. The template to be shown if the condition is true...no?

Reply:
Actually, we just need to remove ! to show the superset and @josehernandezfintecheandomx suggested me, keep it as an environment variable ,but true and false were not working, so I kept it as true in the environment variable and added: !

Reply:
I am not sure i am following you, why the true or false would not work for ngIf which is working based on the variable value is true of false...

Reply:
I know theoretically this should work, but me and @josehernandezfintecheandomx both tried this but it was not working